src/rust/engine/dep_inference/Cargo.toml

```
@@ -9,6 +9,10 @@ publish = false
 tree-sitter = "0.20.10"
 tree-sitter-python = "0.20.2"
 tree-sitter-javascript = "0.20.0"
+checksums = "0.9.1"
+sha2 = "0.10.7"
+walkdir = "2.3.3"
```

These version constraints are upgrading these deps globally (sha2 0.10.6 -> 0.10.7, walkdir 2.3.2 -> 2.3.3)... not necessarily a problem, but potentially better to avoid doing a version upgrade mixed in with other changes?

An alternative would be looser constraints like used in other `Cargo.toml`s:

```suggestion
sha2 = "0.10"
walkdir = "2"
```

---

Ah, huh. Sorry I `cd`d into this dir and did my changes. Didn't realize that'd have global consequences. I'll fix that